docs/content/guides/cell-features/clipboard.md

```diff
@@ -378,6 +277,142 @@ The [`CopyPaste`](@/api/copyPaste.md) plugin exposes the following hooks to mani
+    copyColumnGroupHeaders: true,
+    copyColumnHeadersOnly: true,
+  },
+  colHeaders: true,
+  rowHeaders: true,
+  height: 'auto',
+  nestedHeaders: [
+    ['A', { label: 'B', colspan: 2 }, { label: 'C', colspan: 2 }, { label: 'D', colspan: 2 }, { label: 'E', colspan: 2 }, 'F'],
+    ['G', 'H', 'I', 'J', 'K', 'L', 'M', 'N', 'O', 'P']
+  ],
+  licenseKey: 'non-commercial-and-evaluation'
+});
+```
+:::
+:::
+
+::: only-for react
+::: example #example2 :react
+```jsx
+import { HotTable } from '@handsontable/react';
+import { registerAllModules } from 'handsontable/registry';
+import 'handsontable/dist/handsontable.full.min.css';
+
+// register Handsontable's modules
+registerAllModules();
+
+export const ExampleComponent = () => {
+  return (
+  <HotTable
+    data={[
+      ['A1', 'B1', 'C1', 'D1', 'E1', 'F1', 'G1', 'H1', 'I1', 'J1'],
+      ['A2', 'B2', 'C2', 'D2', 'E2', 'F2', 'G2', 'H2', 'I2', 'J2'],
+      ['A3', 'B3', 'C3', 'D3', 'E3', 'F3', 'G3', 'H3', 'I3', 'J3'],
+      ['A4', 'B4', 'C4', 'D4', 'E4', 'F4', 'G4', 'H4', 'I4', 'J4'],
+      ['A5', 'B5', 'C5', 'D5', 'E5', 'F5', 'G5', 'H5', 'I5', 'J5'],
+    ]}
+    contextMenu={true}
+    copyPaste={{
+      copyColumnHeaders: true,
+      copyColumnGroupHeaders: true,
+      copyColumnHeadersOnly: true,
+    }}
+    colHeaders={true}
+    rowHeaders={true}
+    height="auto"
+    nestedHeaders={[
+      ['A', { label: 'B', colspan: 2 }, { label: 'C', colspan: 2 }, { label: 'D', colspan: 2 }, { label: 'E', colspan: 2 }, 'F'],
+      ['G', 'H', 'I', 'J', 'K', 'L', 'M', 'N', 'O', 'P']
+    ]}
+    licenseKey="non-commercial-and-evaluation"
+      >
+  </HotTable>
+  );
+};
+
+/* start:skip-in-preview */
+ReactDOM.render(<ExampleComponent />, document.getElementById('example2'));
+/* end:skip-in-preview */
+```
+:::
+:::
+
+To add the context menu items, configure the [`CopyPaste`](@/api/copyPaste.md) plugin with these options:
+- [`copyColumnHeaders`](@/api/options.md#copypaste-additional-options)
+- [`copyColumnGroupHeaders`](@/api/options.md#copypaste-additional-options)
+- [`copyColumnHeadersOnly`](@/api/options.md#copypaste-additional-options)
+
+```js
+copyPaste: {
+  copyColumnHeaders: true,
+  copyColumnGroupHeaders: true,
+  copyColumnHeadersOnly: true,
+}
+```
+
+To copy column headers programmatically, call the [`copyPaste.copy()`](@/api/copyPaste.md#copy) method with these arguments:
+- [`'with-column-headers'`](@/api/copyPaste.md#copy)
+- [`'with-all-column-headers'`](@/api/copyPaste.md#copy)
+- [`'column-headers-only'`](@/api/copyPaste.md#copy)
+
+::: only-for react
+::: tip
+To use the Handsontable API, you'll need access to the Handsontable instance. You can do that by utilizing a reference to the `HotTable` component, and reading its `hotInstance` property.
+
+For more information, see the [`Instance Methods`](@/guides/getting-started/react-methods.md) page.
+:::
+:::
+
+```js
+// access the `CopyPaste` plugin instance
+const copyPastePlugin = hot.getPlugin('copypaste');
```

Comment:
It should be const copyPastePlugin = hot.getPlugin('copyPaste');

Reply:
Thanks! Fixed: [6b0bf4b](https://github.com/handsontable/handsontable/pull/10153/commits/6b0bf4b77507467fe152debf1210e8bacc884aaa)